ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -327,6 +327,30 @@ EOF
   echo "Enabled AWS Spot instances for worker nodes"
 }
 
+# enable_efa_pg_instance_config is an AWS specific option that enables one worker machineset in a placement group and with EFA Network Interface Type, other worker machinesets will be ENA Network Interface Type by default.....
+function enable_efa_pg_instance_config() {
+  local dir=${1}
+  #sed -i 's/          instanceType: .*/          networkInterfaceType: EFA\n          placementGroupName: pgcluster\n          instanceType: c5n.9xlarge/' "$dir/openshift/99_openshift-cluster-api_worker-machineset-0.yaml"
+  pip3 install pyyaml --user
+  pushd "${dir}/openshift"
+  python -c '
+import os
+import yaml
+
+for manifest_name in os.listdir("./"):
+    if "machineset" in manifest_name:
+      data = yaml.safe_load(open(manifest_name))
+      data["spec"]["template"]["spec"]["providerSpec"]["value"]["networkInterfaceType"] = "EFA"
+      data["spec"]["template"]["spec"]["providerSpec"]["value"]["instanceType"] = "c5n.9xlarge"
+      data["spec"]["template"]["spec"]["providerSpec"]["value"]["placementGroupName"] = "pgcluster"
+      open(manifest_name, "w").write(yaml.dump(data, default_flow_style=False))
+      print("Patched efa pg into ",  manifest_name)
+      break
+' || exit 2
```

Comment:
Suggest using `return` instead of `exit` in function:
```suggestion
' || return 1
```

Reply:
ok, let me update it.